CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### 🎁 New Features
 
+* Horizontal toolbars will collapse overflowing items into a drop down menu.
```

Comment:
Think we need to highlight the breaking change, that toolbars now cannot be passed in container wrapping components